client/app/scripts/selectors/nodes-chart.js

```diff
@@ -0,0 +1,32 @@
+import { createSelector } from 'reselect';
+import { Map as makeMap } from 'immutable';
+
+
+const allNodesSelector = state => state.get('nodes');
+
+export const nodesSelector = createSelector(
+  [
+    allNodesSelector,
+  ],
+  allNodes => allNodes.filter(node => !node.get('filtered'))
```

Comment:
Great point - that's definitely something to keep an eye on, so I'll add a comment in the code.